mods/cnc/maps/gdi06/map.yaml

```diff
@@ -0,0 +1,1168 @@
+		Location: 48,20
+		Owner: Nod
+		SubCell: 2
+	waypoint27: waypoint
+		Location: 32,32
+		Owner: Neutral
+	Flare: waypoint
+		Location: 30,41
+		Owner: Neutral
+	waypoint17: waypoint
+		Location: 47,22
+		Owner: Neutral
+	waypoint16: waypoint
+		Location: 52,22
+		Owner: Neutral
+	waypoint15: waypoint
+		Location: 54,22
+		Owner: Neutral
+	waypoint14: waypoint
+		Location: 45,22
+		Owner: Neutral
+	waypoint13: waypoint
+		Location: 18,18
+		Owner: Neutral
+	waypoint12: waypoint
+		Location: 20,7
+		Owner: Neutral
+	waypoint11: waypoint
+		Location: 60,10
+		Owner: Neutral
+	waypoint10: waypoint
+		Location: 60,23
+		Owner: Neutral
+	waypoint9: waypoint
+		Location: 38,19
+		Owner: Neutral
+	waypoint8: waypoint
+		Location: 24,18
+		Owner: Neutral
+	waypoint7: waypoint
+		Location: 36,32
+		Owner: Neutral
+	waypoint6: waypoint
+		Location: 49,24
+		Owner: Neutral
+	waypoint5: waypoint
+		Location: 39,13
+		Owner: Neutral
+	waypoint4: waypoint
+		Location: 39,7
+		Owner: Neutral
+	waypoint3: waypoint
+		Location: 26,27
+		Owner: Neutral
+	waypoint2: waypoint
+		Location: 27,32
+		Owner: Neutral
+	TransportRally: waypoint
+		Location: 36,42
+		Owner: Neutral
+	UnitsRally: waypoint
+		Owner: GDI
+		Location: 55,52
+	lstEnd: waypoint
+		Owner: GDI
+		Location: 55,53
+	lstStart: waypoint
+		Owner: GDI
+		Location: 55,61
+
+Smudges:
+
+Rules:
+	World:
+		-SpawnMPUnits:
+		-MPStartLocations:
+		-CrateSpawner:
+		LuaScript:
+			Scripts: gdi06.lua
+		ObjectivesPanel:
+			PanelName: MISSION_OBJECTIVES
+		MusicPlaylist:
+			BackgroundMusic: ambientrain
+		WeatherOverlay:
+			ParticleDensityFactor: 0.0007625
+			ChangingWindLevel: true
+			WindLevels: -5, -3, -2, 0, 2, 3, 5
+			WindTick: 150, 550
+			InstantWindChanges: false
+			UseSquares: false
+			PaticleSize: 0, 0
+			ScatterDirection: 0, 0
+			Gravity: 8.00, 12.00
+			SwingOffset: 0, 0
+			SwingSpeed: 0, 0
+			SwingAmplitude: 0, 0
+			ParticleColors: 255,48,64,116, 255,40,56,108, 255,32,44,96, 255,24,44,84
+			LineTailAlphaValue: 150
+		GlobalLightingPaletteEffect:
+			Red: 0.75
+			Green: 0.85
+			Blue: 1.5
+			Ambient: 0.35
+	Player:
+		-ConquestVictoryConditions:
+		MissionObjectives:
+			EarlyGameOver: true
+	^Vehicle:
+		Tooltip:
+			GenericVisibility: Enemy
+			ShowOwnerRow: false
+	^Tank:
+		Tooltip:
+			GenericVisibility: Enemy
+			ShowOwnerRow: false
+	^Helicopter:
+		Tooltip:
+			GenericVisibility: Enemy
+			ShowOwnerRow: false
+	^Infantry:
+		Tooltip:
+			GenericVisibility: Enemy
+			ShowOwnerRow: false
+	^Plane:
+		Tooltip:
+			GenericVisibility: Enemy
+			ShowOwnerRow: false
+	^Ship:
+		Tooltip:
+			GenericVisibility: Enemy
+			ShowOwnerRow: false
+	^Building:
+		Tooltip:
+			GenericVisibility: Enemy
+			ShowOwnerRow: false
+	^Wall:
+		Tooltip:
+			ShowOwnerRow: false
+	FLARE:
+		RevealsShroud:
+			Range: 5c0
+	OLDLST:
+		Inherits: LST
+		-WithRoof:
+		-Selectable:
+		RejectsOrders:
+	SAM:
+		SpawnActorOnDeath:
+			Actor: e1
+	RMBO:
+		MustBeDestroyed:
+			RequiredForShortGame: true
```

Comment:
the ShortGame settings are used so that the mission ends as soon as the RMBO dies (even if the Chinook is still alive) - without it, the mission would continue with the Chinook